runner/src/server/views/payment-skip-warning.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'layout.html' %}
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-caption-l govuk-!-margin-top-0">
+      {{customText.caption if customText.caption else 'Payment'}}</h2>
+    <h1 class="govuk-heading-l">
+      {{customText.title}}
+    </h1>
+    <div class="govuk-body">
+      {% if customText.body %}
+        {{ customText.body | safe }}
+      {% else %}
+        Only continue if you are unable to pay online, because for example you do not have a debit or credit card.
```

Comment:
Do we need the "because" in here? I know it's not a huge issue because people can just specify their own custom text but :shrug: